geode-docs/configuring/cluster_config/export-import.html.md.erb

```diff
@@ -51,6 +51,8 @@ import cluster-configuration --zip-file-name=/home/username/configs/myClusterCon
 See [import cluster-configuration](../../tools_modules/gfsh/command-pages/import.html#topic_vnv_grz_ck).
 
 **Note:**
-You cannot import a cluster configuration to a cluster where cache servers are already running.
-
-
+Data members (servers) obtain the new cluster configuration when they start.
+You can import a cluster configuration to a cluster where cache servers are already running,
```

Comment:
change it to something like "you can import a cluster configuration to a cluster only when there is no running cache servers or the only running cache servers are those that do not have any regions defined in them. If you have running servers that have any regions already defined, the import will fail".